recipes/matplotlib-fontja/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "matplotlib-fontja" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/matplotlib_fontja-{{ version }}.tar.gz
+  sha256: d3b92a1927a9c1dad2932423668777742b6b3f9172b33fe86300dc5bbc2755e6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling
+    - pip
+  run:
+    - python >=3.7
+    - matplotlib-base
+
+test:
+  imports:
+    - matplotlib_fontja
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ciffelia/matplotlib-fontja
+  summary: matplotlibを日本語表示に対応させます。
```

Comment:
Can you please add an English summary. OK to have both but we have English as a basis here and it would be good if core is able to understand it.

Reply:
Thank you for your review and comment.

I added the English summary. (7a5a4cb73818ce5b6b85430e0dd8702f0e0b80dd)
Please review it whenever you have time.